File: Data Structure/Lab 3/ReadArr.c
// Write a program to read and display n numbers using an array.

#include<stdio.h>
void main(){
    int n,arr[n],i=0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&arr[i]);
    }

    for(i = 0; i < n ; i++){
        printf("%d ,",arr[i]);
    }
}

File: Data Structure(Java)/Lab 2/ReadArr.java
// Write a program to read and display n numbers using an array.

import java.util.Scanner;

public class ReadArr {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter n :");
        int n = sc.nextInt();
        int[] arr = new int[n];

        for(int i = 0; i < n; i++){
            System.out.println("Enter number at "+i);
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < n; i++){
            if(i == n - 1){
                System.out.print(arr[i]);
            }
            else{
                System.out.print(arr[i]+",");
            }
        }

        sc.close();
    }
}
